api/GovStack_Registration_BB_API.json

```diff
@@ -60,14 +58,12 @@
 		},
 		"/services/{serviceKey}": {
 			"get": {
-				"tags": [
-					"Applicant"
-				],
+				"tags": ["Available Services"],
 				"summary": "returns detailed information about a service",
 				"description": "Pass in the ID of the service and it will return all information about that service\n",
 				"operationId": "getService",
 				"parameters": [{
-					"$ref": "#/components/parameters/serviceIdParam"
+					"$ref": "#/components/parameters/serviceKeyParam"
```

Comment:
let's make naming more consistent - maybe best use `serviceId`, which is more in line with the other parameters' naming?

Reply:
Fine with me